.github/workflows/release-build.yml

```diff
@@ -22,20 +24,18 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 8
+      - name: Print output every minute to avoid Github Actions timeout
```

Comment:
when job stop print output by sometime (I don't know how long) it will be hang and kill after timeout

Reply:
Hmmm, do you have a reference for this?

Reply:
https://github.com/mockito/mockito-testng/issues/26

Reply:
Thanks for the link!

These issues seem to be related to Travis CI, not GitHub Actions. For instance, [this issue](https://github.com/bintray/gradle-bintray-plugin/issues/240) or [this suggestion](https://github.com/mockito/mockito-testng/pull/35#issuecomment-762293490). From the [Travis CI docs](https://docs.travis-ci.com/user/common-build-problems/#my-builds-are-timing-out):

>Our builds have a global timeout and a timeout that's based on the output. If no output is received from a build for 10 minutes, it's assumed to have stalled for unknown reasons and is subsequently killed.

I couldn't find anything similar when it comes to GitHub Actions.

Reply:
It also occurs on GitHub Actions
```
2020-12-07T10:12:08.7002417Z Uploading to https://api.bintray.com/content/mockito/maven/mockito-testng/0.1.5/org/mockito/mockito-testng/0.1.5/mockito-testng-0.1.5-sources.jar...
2020-12-07T10:12:38.6983788Z 
2020-12-07T10:12:38.6985346Z > Task :bintrayPublish
2020-12-07T16:11:35.3475891Z ##[error]The operation was canceled.
```

And with workaround
```
2021-01-18T22:29:46.6942806Z > Task :bintrayPublish
2021-01-18T22:30:40.9698429Z =====[ 120 seconds elapsed -- still running ]=====
2021-01-18T22:31:40.9711465Z =====[ 180 seconds elapsed -- still running ]=====
2021-01-18T22:32:12.2676011Z 
```

Reply:
Hmmm, isn't this because between:

```
...10:12:38...
...16:11:35...
```

Almost 6 hours elapsed, timeout kicks in. Whereas here:

```
...22:29:46...
...22:32:12...
```

Only a few minutes elapsed.

If the upload takes too long – for whatever reason(s) – it doesn't matter whether the job outputs something. It will be canceled after 360 minutes / 6 hours.

What do you think?

Reply:
Ok, I will remove it .. we will see if it was need, maybe not

Reply:
Thank you, let's hope so. 😉